### ISSUE TITLE
Mod shows its loaded, but doesn't do anything

### ISSUE THREAD

**Issue by daglamier22** [chaorace/cqui#316]
When i hit Esc in game i can see the mod in the list of loaded DLC, but it doesn't appear to be doing anything. Main thing i was looking for was the map pins and the leader info at the top right of the screen, but they are all standard. I am on Linux and using the latest nightly build, which appears to have a previous Linux related fix.

I can't seem to find any kind of game logs, do they exist for troubleshooting mods?

**Comment by chaorace** [comment]
Just to be sure, are you starting a new game with CQUI enabled or attempting to use it with an existing save? Once you start a game, it won't load new mods even if you install them and they look enabled

**Comment by daglamier22** [comment]
Yes, i was trying new games.

**Comment by chaorace** [comment]
Ok, and just to be sure, are you using the latest nightly?

**Comment by chaorace** [comment]
@daglamier22 Hey, could you try the [Linux release over on the Steam Workshop](http://steamcommunity.com/sharedfiles/filedetails/?id=871529854) and tell me if that fixes your issue?

**Comment by chaorace** [comment]
@daglamier22 I'll be closing this issue in 24 hours if I no response. If you come back and find this issue has been closed, please respond here so I can reopen it

**Comment by daglamier22** [comment]
I had to leave town unexpectedly for a funeral. Will test that version and respond on Monday when I get home

**Comment by chaorace** [comment]
@daglamier22 My apologies, take your time. I will patiently await a better time when you're available

**Comment by daglamier22** [comment]
Sorry about that, I'm back in town. I removed the previous version I downloaded and instead installed the OSX/Linux version from the workshop. It seems most of it works now. The only thing that doesn't seem to be working is the Leader icons at the top right of the screen. I don't even get the default icons, there's just nothing there at all. I even played until i met an AI and no change. Any ideas?

**Comment by daglamier22** [comment]
was just playing through a game and found that the civics tree also will not open, right clicking the minimap doesn't change its size, it seems to be stuck in the larger size (not a big deal though), and the leader icons at the top right also still aren't appearing at all

**Comment by daglamier22** [comment]
I had some progress on this just a few moments ago. I went into diplomacyribbon.lua and civicstree.lua and commented out some code; the "Team Ribbon" and "unlockEnvoy" sections respectively.

I think there might be a versioning problem in the workshop. I subscribed to the OSX/Linux version in the workshop but the files don't match the OSX-Linux Branch here on github, they match the master branch. I noticed the errors in my Lua.log were pointing to lines that exist in the master branch but not the linux branch. Commenting them out has both the diplomacy ribbon and the civics tree working perfectly.
